db/schema.rb

```diff
@@ -175,6 +160,21 @@
 
   add_index "roles", ["name"], name: "roles_name_idx", unique: true, using: :btree
 
+  create_table "steps", force: :cascade do |t|
+    t.integer  "user_id"
+    t.string   "status",              limit: 255
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.integer  "position"
+    t.integer  "proposal_id"
+    t.datetime "approved_at"
```

Comment:
There are still a load of methods and things using `Approval`/`approver` language in this PR; we're not changing it all in one go. And we still need to work out what all the new terminology should be. `completed_at`? `performed_at`?